mcda/utils/utils_for_plotting.py

```diff
@@ -226,11 +236,19 @@ def save_figure(figure: object, folder_path: str, filename: str):
     :param filename: str
     :return: None
     """
+    custom_output_path = os.environ.get('PROMCDA_OUTPUT_DIRECTORY_PATH')  # check if an environmental variable is set
+    output_directory_path = custom_output_path if custom_output_path else DEFAULT_OUTPUT_DIRECTORY_PATH
+
     timestamp = datetime.now().strftime("%Y-%m-%d_%H-%M-%S")
     new_filename = f"{timestamp}_{filename}"
 
-    result_path = os.path.join(folder_path, new_filename)
-    figure.write_image(result_path)
+    full_output_path = os.path.join(output_directory_path, folder_path, new_filename)
+    try:
+        utils_for_main.ensure_directory_exists(full_output_path)
+    except IOError:
```

Comment:
This is of no use as long as the method being called already implements it. See comment above.